specification/resources/resource-manager/Microsoft.Resources/preview/2022-08-01-preview/deploymentStacks.json

```diff
@@ -0,0 +1,1353 @@
+        "x-ms-examples": {
+          "DeploymentStacksExportTemplate": {
+            "$ref": "./examples/DeploymentStackSubscriptionExportTemplate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/DeploymentStackNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the Template or TemplateLink payload of the deployment stack.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStackTemplateDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStacksError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.Resources/deploymentStacks/{deploymentStackName}/exportTemplate": {
+      "post": {
+        "tags": [
+          "DeploymentStacks"
+        ],
+        "operationId": "DeploymentStacks_ExportTemplateAtManagementGroup",
+        "description": "Exports the template used to create the deployment stack.",
+        "x-ms-examples": {
+          "DeploymentStacksExportTemplate": {
+            "$ref": "./examples/DeploymentStackManagementGroupExportTemplate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/DeploymentStackNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the Template or TemplateLink payload of the deployment stack.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStackTemplateDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentStacksError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeploymentStack": {
+      "type": "object",
+      "description": "Deployment stack object.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AzureResourceBase"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
```

Comment:
@harshpatel17  the reason you are getting the TrackedREsourcePathc error is becasue of this location. 

I thought you said that this is a proxy resource if so why are you definitng it with a location ?

Reply:
@raosuhas Hey Suhas! We (the deployments team) have added ARM code to support location for proxy resources.  (It relies on a cosmos DB in ARM under-the-hood to route proxy resource requests to the proper location).  We called them ProxyLocationMappedResources, if you want to investigate further in the ARM codebase. For now, it's only for deployments and deployment stacks.